### PR TITLE
Add SDK

### PR DESCRIPTION
**Pull request by lenzfliker**
Resolved #21 @eldadfux

### CODE REVIEW

**Review by notnavindu — APPROVED** [review]

**Comment by lenzfliker** [comment]
@eldadfux

File: README.md [hunk 1]
```diff
@@ -92,6 +94,17 @@ Share your apps here! send a pull request!
 * [Create a collection with Node.js](https://gist.github.com/eldadfux/0591d2205311b02966e5e365fbafad2a)
 * [Delete Appwrite Docker containers and volumes](https://gist.github.com/PedroCisnerosSantana/618a088f60a64f227363c6c10e80b366)
 
+## SDK
+
+* [Appwrite Web SDK](https://github.com/appwrite/sdk-for-js)
+* [Appwrite Flutter SDK](https://github.com/appwrite/sdk-for-flutter)
```
Comment (eldadfux) [inline README.md:100]: Please add ` (beta)` with space after `[Appwrite Flutter SDK`

File: README.md [hunk 2]
```diff
@@ -92,6 +94,17 @@ Share your apps here! send a pull request!
...
+* [Appwrite PHP SDK](https://github.com/appwrite/sdk-for-php)
+* [Appwrite Node.js SDK](https://github.com/appwrite/sdk-for-node)
+* [Appwrite SDK for Go](https://github.com/appwrite/sdk-for-go)
```
Comment (eldadfux) [inline README.md:103]: The GO SDK is still a work in progress, please remove it.

File: README.md [hunk 3]
```diff
@@ -92,6 +94,17 @@ Share your apps here! send a pull request!
...
+* [Appwrite Python SDK](https://github.com/appwrite/sdk-for-python)(beta)
```
Comment (eldadfux) [inline README.md:104]: Please add ` (beta)` with space after `[Appwrite Python SDK`

File: README.md [hunk 4]
```diff
@@ -92,6 +94,17 @@ Share your apps here! send a pull request!
...
+* [Appwrite Ruby SDK](https://github.com/appwrite/sdk-for-ruby)(beta)
```
Comment (eldadfux) [inline README.md:105]: Please add ` (beta)` with space after `[Appwrite Ruby SDK`

File: README.md [hunk 5]
```diff
@@ -92,6 +94,17 @@ Share your apps here! send a pull request!
...
+* [Appwrite Deno SDK](https://github.com/appwrite/sdk-for-deno)(beta)
```
Comment (eldadfux) [inline README.md:106]: Please add ` (beta)` with space after `[Appwrite Deno SDK`

**Review by eldadfux — CHANGES_REQUESTED** [review]

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
THANK YOU! All changes merged 🥳

Please reach out to me on our [Discord server](https://appwrite.io/discord) if you would like me to send you some cool Appwrite stickers and magnets! Please note that shipping might take longer times because of COVID19.

As a way of saying thank you, we have just invited you to join the Appwrite organization on GitHub. You can accept the invite by visiting https://github.com/orgs/appwrite/invitation. By joining our team, you will officially be an Appwrite maintainer on GitHub.

You can change your membership [visibility settings](https://help.github.com/en/github/setting-up-and-managing-your-github-user-account/publicizing-or-hiding-organization-membership), so your new Appwrite team membership badge will show up on your personal GitHub profile.

**Pull request merged**